rest_api.py

```diff
@@ -1,11 +1,13 @@
 """REST API."""
-
-import json
-from file_adapter import FileAdapter
-from filter import filter_releases
-from filter import sort
-from flask_restful import reqparse
-from flask_restful import Resource
+import sys
+sys.path.append('../')
```

Comment:
why is this required ?

Reply:
Can you do https://stackoverflow.com/questions/72852/how-to-do-relative-imports-in-python instead?